src/base/b-virtual-scroll/b-virtual-scroll.ts

```diff
@@ -173,35 +124,23 @@ export default class bVirtualScroll extends iData {
 	@prop({type: [Object, Array], required: false})
 	readonly request?: RequestParams;
 
+	/**
+	 * Engine for request remote data
+	 */
+	@prop({type: Function, default: (ctx, query) => ctx.get(query), required: false})
+	readonly requestEngine!: RequestEngine;
+
 	/**
 	 * If, when calling a function, it returns true, then the component will be able to request additional data
```

Comment:
When this function returns true the component will be able to request additional data